real_estate/models/estate_property_type.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+class EstatePropertyType(models.Model):
+
+    _name = 'estate.property.type'
+    _description = 'Detail About Perticular Property'
+
+    name = fields.Char(
+        string='property Type',
```

Comment:
```suggestion
        string='Name',
```
Don't you think it will be a name for `Property Type` itself?